src/iris/api.py

```diff
@@ -4243,18 +4243,68 @@ def on_get(self, req, resp):
             stats = app_stats.calculate_global_stats(connection, cursor, fields_filter=fields_filter)
 
         else:
-            cursor.execute('SELECT `statistic`, `value` FROM `global_stats`')
+            cursor.execute('SELECT `statistic`, `value`, `timestamp` FROM `global_stats`')
             if cursor.rowcount == 0:
                 logger.exception('Error retrieving global stats from db')
                 raise HTTPInternalServerError('Error retrieving global stats from db')
-            stats = {row[0]: row[1] for row in cursor}
+
+            stats = {}
+
+            for row in cursor:
+                # {statistic : {timestamp: value, timestamp: value}}
+                if stats.get(row[0]):
+                    stats[row[0]][row[2]] = row[1]
+                else:
+                    stats[row[0]] = {}
+                    stats[row[0]][row[2]] = row[1]
 
         cursor.close()
         connection.close()
         resp.status = HTTP_200
         resp.body = ujson.dumps(stats, sort_keys=True)
 
 
+class HighPriorityIncidents(object):
```

Comment:
I considered handling as /stats/{statistic} but since high priority incidents per week was the only statistic highlighted as being of particular importance for weekly reporting I decided to avoid naming conflicts and just make it its own endpoint.